<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>712</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>计时小工具</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/ico/date.ico</normaloff>:/ico/date.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,1,1,5">
     <item>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="digitCount">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;仿宋&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>请设置当前时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_h">
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;仿宋&quot;;
color: rgb(170, 85, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>16</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;仿宋&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>时</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_m">
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;仿宋&quot;;
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>16</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;仿宋&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>分</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_s">
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;仿宋&quot;;
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>16</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;仿宋&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>秒</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_start">
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;仿宋&quot;;

color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_stop">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>16</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;仿宋&quot;;

color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_pause">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>16</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;仿宋&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_log">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>16</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;仿宋&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>打点</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
font: 75 16pt &quot;仿宋&quot;;</string>
         </property>
         <property name="text">
          <string>修改秒数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_change">
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;仿宋&quot;;
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_add">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>16</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;仿宋&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>增加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_reduce">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>16</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;仿宋&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>减少</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
